apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -95,9 +95,51 @@ const EVENTS = {
   AGE_21_SELECTED_EVENT: 'Age 21+ Selected',
   GUIDE_SENT_EVENT: 'Guide Sent',
 
+  // Hour of Code - Dance Party
+  DANCE_PARTY_ACTIVITY_STARTED: 'Dance Party Activity Started',
```

Comment:
minor: could we combine this with `EVENT_GROUPS` below into something like
```
const EVENTS = {
  DANCE_PARTY_ACTIVITY_STARTED: {
    name: 'Dance Party Activity Started',
    group: GROUP_NAMES.DANCE_PARTY
  },
  DANCE_PARTY_LEVEL_COMPLETED: { ... },
}
```
So that constants and the mapping is declared once? Though, I'm realizing that `EVENTS` is already quite large so that may end up being quite a refactor, so maybe not...